### ISSUE TITLE
`VkAccessFlagBits` broken link

### ISSUE THREAD

**Issue by Dherse** [KhronosGroup/Vulkan-Docs#1723]
Hello,

In the documentation for [`VkAccessFlagBits`](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkAccessFlagBits.html) there is a broken link: at line 1086 for `synchronization.txt` (note I'm using an older commit so this may be slightly incorrect), the link that gets generated with the text `color, resolve, or depth/stencil resolve attachment` gives the link `#renderpass` instead of `https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#renderpass`.

Hope this is helpful,

Dherse

**Comment by oddhack** [comment]
Thanks. I'll take a look at it once things start up again after the holidays.

**Comment by Dherse** [comment]
I am adding onto this issue that there are similar problems in `VkBufferMemoryBarrier.html` where two links with texts `[synchronization-queue-transfers]` also have a broken link being only `#synchronization-queue-transfers`.

Happy holidays,

Dherse

**Comment by Dherse** [comment]
Hello,

I found another example, in `PFN_vkReallocationFunction.html` and others, the link `vkAllocationFunction_return_rules.html` is broken.

There is also something very weird with the generation of `PFN_vkInternalFreeNotification.html` for the description section, where for some reason it generated a link in the description but puts it in a `<code></code>` block. The link cannot be clicked and has unusual text for the Vulkan docs.

Regards,

Dherse

**Comment by oddhack** [comment]
You're correct that the refpage extraction process doesn't always rewrite links correctly. It does a pretty good job with the actual refpages blocks, with a few exceptions, but the common validity statements exist in separate asciidoc include files and xrefs there that *should* be rewritten, aren't. It is a known issue but not a high priority.

**Comment by oddhack** [comment]
N.b. what we'd probably have to do is change the refpage extraction scripts to actually insert the text of those xrefs, rather than incorporate them via the 'include::' construct, prior to xref rewriting. It is not hard, but tedious.

**Comment by oddhack** [comment]
We are in process of migrating the refpages to docs.vulkan.org, which will provide a much better experience and fix most, if not all of these issues. For example if you go to https://docs.vulkan.org/refpages/latest/refpages/source/VkPhysicalDeviceSubgroupProperties.html the links to sections of the spec point out correctly to the spec. I'm going to close this as fixed, in this context, and sometime in the next month or so we'll switch over the refpages in the Vulkan registry to rewrite to docs.vulkan.org instead. So, closing this. There are further improvements coming on the docs site, in particular the loading time issue with the huge search index should be fixed when I next update the site in a week or so, but those are not content-related.
